incubator/airflow/templates/configmaps.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ template "airflow.fullname" . }}-env"
+data:
+  ## Force UTC timezone
+  TZ: Etc/UTC
+  ## Postgres DB configuration
+  POSTGRES_HOST: "{{ template "airflow.postgresql.fullname" . }}"
+  POSTGRES_DB: "{{ .Values.postgresql.postgresDatabase }}"
+  POSTGRES_USER: "{{ .Values.postgresql.postgresUser }}"
+  POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresPassword }}"
+  ## Redis DB configuration
+  REDIS_HOST: "{{ template "airflow.redis.fullname" . }}"
+  REDIS_PASSWORD: "{{ .Values.redis.redisPassword }}"
```

Comment:
Again Passwords in a ConfigMap are not a good idea.

Reply:
todo after merge